File: Linguagem C/1012.c
#include <stdio.h>

const double pi = 3.14159;
double a = 0, b = 0, c = 0;
double triangulo = 0, circulo = 0, trapezio = 0, quadrado = 0, retangulo = 0;

int main(){
	scanf("%lf",&a);
	scanf("%lf",&b);
	scanf("%lf",&c);
	
	triangulo = (a * c)/2;
	printf("TRIANGULO: %.3lf\n",triangulo);
	
	circulo = pi * (c*c);
	printf("CIRCULO: %.3lf\n",circulo);
	
	trapezio = ((a + b) / 2)*c;
	printf("TRAPEZIO: %.3lf\n",trapezio);
	
	quadrado = b * b;
	printf("QUADRADO: %.3lf\n",quadrado);
	
	retangulo = a * b;
	printf("RETANGULO: %.3lf\n",retangulo);
	
	return 0;
}

File: Linguagem Java/1012.java
package pkg1012;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        
        double a = input.nextDouble();
        double b = input.nextDouble();
        double c = input.nextDouble();
        
        double areaTriangulo = (a*c)/2;
        double areaCirculo = 3.14159 * (c*c);
        double areaTrapezio = ((a+b)*c)/2;
        double areaQuadrado = b*b;
        double areaRetangulo = a*b;
        
        System.out.printf("TRIANGULO: %.3f\n", areaTriangulo);
        System.out.printf("CIRCULO: %.3f\n", areaCirculo);
        System.out.printf("TRAPEZIO: %.3f\n", areaTrapezio);
        System.out.printf("QUADRADO: %.3f\n", areaQuadrado);
        System.out.printf("RETANGULO: %.3f\n", areaRetangulo);
    
        
    }
    
}
